bigtop-packages/src/common/apex/do-component-build

```diff
@@ -28,6 +28,9 @@ command -v mvn > /dev/null 2>&1 || {
 # Build apex
 mvn clean package -DskipTests "$@"
 
+#Find hadoop.version from pom.xml
+APEX_HADOOP_VERSION=$(grep -oPm1 "(?<=<hadoop.version>)[^<]+" pom.xml)
```

Comment:
Apex min hadoop version is 2.6.0. I believe if I use hadoop version from bigtop.bom, that would give me 2.7.3. What I want to do here is, find out the min version of hadoop which apex uses and exclude those from packaging.

Hence done this way.